dbt/tests/data_tests/generic_tests/expect_unimputed_values_are_unchanged.sql

```diff
@@ -0,0 +1,18 @@
+{% test expect_unimputed_values_are_unchanged(
+    model,
+    original,
+    imputed,
+    imputation_code,
+    atol=0.1,
+    rtol=0.001
+) %}
+SELECT * FROM {{ model }}
+WHERE {{ imputation_code }} IS NULL
+AND (
+    (abs({{ original }} - {{ imputed }}) > {{ atol }})
+    OR (abs({{ original }} - {{ imputed }}) / greatest(abs({{ original }}), abs({{ imputed }})) > {{ rtol }})
```

Comment:
Does the math and default values for rtol / atol here seem reasonable?

Reply:
Looks good to me!